chia/pools/pool_wallet.py

```diff
@@ -892,9 +892,9 @@ async def new_peak(self, peak_height: uint64) -> None:
 
         if self.target_state is None:
             return
-        if self.target_state == pool_wallet_info.current.state:
```

Comment:
was this just a typo? is there no field `state` in `pool_wallet_info.current`?

Reply:
It was a typo. There is a field state in both, but what was intended is to compare the whole `PoolState`, NOT `PoolState.state`, which is a  `PoolSingletonState`. Better naming would help here.

The state machine for `PoolSingletonState` is very simple, but two `PoolState`s with different members are not equivalent.